.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
Does changing the formatting engine need to be in this PR? Seems like it's adding a lot.

Proposal: split it out so I can evangelize Prettier? 😄

Reply:
Personally a fan of prettier as well, so I'm fine with doing it in a separate PR with prettier! 💅

Reply:
I mean, I am personally up for it - so our editors know what way to follow.